Configure

```diff
@@ -898,7 +900,7 @@ while (@argvcopy)
                 {
                 $guess_opts{verbose} = 1;
                 }
-        elsif (/^-w$/)          # From older 'config'
```

Comment:
Removing only one instance of this comment?  Then why remove it at all?

Reply:
Because I only documented that one.